apps/system/utils/fscmd.c

```diff
@@ -1336,6 +1337,80 @@ static int tash_df(int argc, char **args)
 }
 #endif
 
+#ifndef CONFIG_DISABLE_ENVIRON
+#if !defined(CONFIG_DISABLE_MOUNTPOINT) && \
+	!defined(CONFIG_DISABLE_PSEUDOFS_OPERATIONS) && defined(CONFIG_BCH)
+	
+enum fs_minor_num_e {
+	FS_BLOCK_MINOR_PRIMARY = 0, // minor number primary block driver, usually for /mnt
+	FS_BLOCK_MINOR_SECONDARY = 1 // usually for /ext
+};
+typedef enum fs_minor_num_e fs_minor_t;
+
+static int format_filesystem(fs_minor_t minor)
+{
+	char name[CONFIG_PATH_MAX];
+	int fd;
+	int ret = ERROR;
+
+	if ((minor < FS_BLOCK_MINOR_PRIMARY) || (minor > FS_BLOCK_MINOR_SECONDARY)) {
+		printf("invalid Minor number : %d", minor);
+		return ERROR;
+	}
+	for (int i = 0; i < 9; i++) {
```

Comment:
Could you make define for `9`? Because it is not easy to understand why it is 9.

Reply:
That policy should be defined in partition level, currently we don't have a MAX number of partitions,
so we should guide partition of smartfs should be located in front area of partition list. (under 9)